mdoc-sbt/src/main/scala/mdoc/MdocPlugin.scala

```diff
@@ -90,7 +90,7 @@ object MdocPlugin extends AutoPlugin {
           parsed
         ).flatten.mkString(" ")
         Def.taskDyn {
-          runMain.in(Compile).toTask(s" mdoc.Main $args")
+          runMain.in(Compile).toTask(s" mdoc.SbtMain $args")
```

Comment:
Would you be able to add a scripted test to check the regression?

Reply:
Sure, whichever is fine. Maybe it's better since it indicates that we are talking about main that should be used for a build tool.

Reply:
add scaladoc on top of it + a scripted test that checks `--check` and `--check-link-hygiene` do not exit but still fail sbt task